docs/source/get_started/access_info/access_info_css.ipynb

```diff
@@ -84,7 +84,7 @@
     {
```

Comment:
<p>I think we can add this <code>qss</code> notebook note: "Note this returns a list and you must specify the index if you want to retrieve a single circuit." here as well (see &nbsp;<a href="https://github.com/Infleqtion/client-superstaq/pull/743" target="_blank">https://github.com/Infleqtion/client-superstaq/pull/743</a>) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/728/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>